vibes/vibed-ui/src/core/Application.cpp

```
@@ -0,0 +1,71 @@
+            }
+        }
+
+        if (!running) {
+            break;
+        }
+
+        // TODO: Rendering.
+    }
+}
+
+void Application::setWindowSize(int w, int h) {
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'setWindowSize' of similar type ('int') are easily swapped by mistake

(bugprone-easily-swappable-parameters)